src/AI/CoreLogic.pm

```diff
@@ -1028,18 +1028,16 @@ sub processTransferItems {
 		foreach ( [ $row->{source}, $source ], [ $row->{target}, $target ] ) {
 			my ( $name, $list ) = @$_;
 			next if $list->isReady;
-			error T( "Your inventory is not available. Unable to transfer item '%s'.\n", $row->{item} ) if $name eq 'inventory';
-			error T( "Your storage is not available. Unable to transfer item '%s'.\n",   $row->{item} ) if $name eq 'storage';
-			error T( "Your cart is not available. Unable to transfer item '%s'.\n",      $row->{item} ) if $name eq 'cart';
+			#name can be storage, inventory or cart
+			error T( "Your $name is not available. Unable to transfer item '%s'.\n", $row->{item} );
```

Comment:
Interpolation with T (translation method) is forbidden.
It should go like this:
```suggestion
			error TF( "Your %s is not available. Unable to transfer item '%s'.\n", $name, $row->{item} );
```

Now that I look at it, the old way is also wrong, and should be using TF instead of T